src/evm_transition_tool/transition_tool.py

```diff
@@ -571,7 +590,13 @@ def evaluate(
         if int(env["currentNumber"], 0) == 0:
             reward = -1
         t8n_data = TransitionTool.TransitionToolData(
-            alloc=alloc, txs=txs, env=env, fork_name=fork_name, chain_id=chain_id, reward=reward
+            alloc=alloc,
+            txs=txs,
+            env=env,
+            fork_name=fork_name,
+            chain_id=chain_id,
+            reward=reward,
+            empty_string_to=self.empty_string_to(),
```

Comment:
A part of me feels like it would be better for the evmone t8n to handle this, as this maybe bloats `t8n_data` with a field that isn's "super" important in comparison to the others.

I think it could be worth exploring, but very happy with this as a workaround :D

Reply:
I agree, both tools should expect the same type of input.

Filed an issue in the evmone repo for this: https://github.com/ethereum/evmone/issues/923

Should we merge as is and rollback once the behavior is standardized?

Reply:
Lets do it! :D